### PR TITLE
feat(components): adjust ui contract and account detail

### PR DESCRIPTION
**Pull request by jennieramida**
## Describe your changes

fixes: CFE-502, CFE-503, CFE-504, CFE-505, CFE-506, CFE-506, CFE-507

### CODE REVIEW

File: src/lib/components/state/AccountDetailEmptyState.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import type { FlexProps, TextProps } from "@chakra-ui/react";
+import { Flex, Text } from "@chakra-ui/react";
+
+export interface AccountDetailEmptyStateProps {
+  message: string;
+  textVariant?: TextProps["variant"];
+  pt?: FlexProps["pt"];
+}
+
+export const AccountDetailEmptyState = ({
```
Comment (songwongtp) [inline src/lib/components/state/AccountDetailEmptyState.tsx:10]: move to `account-details/components/` and rename both filename and the component name to `AccountDetailsEmptyState`

File: src/lib/components/table/TableTitle.tsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ interface TableTitleProps extends BoxProps {
   title: string;
   count?: number;
   showCount?: boolean;
-  helperText?: string;
+  helperText?: string | undefined;
```
Comment (songwongtp) [inline src/lib/components/table/TableTitle.tsx:8]: remove `| undefined` already have `?`

File: src/lib/pages/account-details/components/tables/txs/Full.tsx [hunk 3]
```diff
@@ -4,28 +4,59 @@ import { useEffect, useMemo, useState } from "react";
...
+  selectedFilters,
+}: {
+  transactions:
+    | {
+        items: Transaction[];
       }
-    : {
-        message: "No transactions have been submitted by this account before.",
-      };
+    | undefined;
```
Comment (songwongtp) [inline src/lib/pages/account-details/components/tables/txs/Full.tsx:31]: `transactions: Option<Transaction[]>`

File: src/lib/pages/account-details/components/tables/txs/Full.tsx [hunk 4]
```diff
@@ -141,16 +172,7 @@ export const TxsTableFull = ({
...
-                !transactions ? (
-                  <ErrorFetching dataName="transactions" />
-                ) : (
-                  <EmptyState
-                    withBorder
-                    {...getEmptyStateProps(selectedFilters)}
-                  />
-                )
-              }
+              emptyState={getEmptyState({ transactions, selectedFilters })}
```
Comment (songwongtp) [inline src/lib/pages/account-details/components/tables/txs/Full.tsx:175]: `getEmptyState({ transactions?.items, selectedFilters })`

**Review by songwongtp — CHANGES_REQUESTED** [review]

**Review by songwongtp — APPROVED** [review]

**Review by evilpeach — APPROVED** [review]

**Pull request merged**